types/c3/index.d.ts

```diff
@@ -1,26 +1,57 @@
-// Type definitions for C3js 0.7
+// Type definitions for c3 0.7
 // Project: http://c3js.org/, https://github.com/c3js/c3
 // Definitions by: Marc Climent <https://github.com/mcliment>
 //                 Gerin Jacob <https://github.com/gerinjacob>
 //                 Bernd Hacker <https://github.com/denyo>
 //                 Dzmitry Shyndzin <https://github.com/dmitryshindin>
 //                 Tim Niemueller <https://github.com/timn>
 //                 Nate Mara <https://github.com/natemara>
+//                 Ian Sanders <https://github.com/iansan5653>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.5
```

Comment:
Do you intend for this to only work in TypeScript 3.5+?

Reply:
That should be 3.0, because I used `unknown`. Thanks for the heads up